service/controller/key/testdata/case-2-master-render.golden

```diff
@@ -519,107 +519,4 @@ storage:
       contents:
         source: "data:text/plain;charset=utf-8;base64,YXBpVmVyc2lvbjogdjEKa2luZDogQ29uZmlnCnVzZXJzOgotIG5hbWU6IHNjaGVkdWxlcgogIHVzZXI6CiAgICBjbGllbnQtY2VydGlmaWNhdGU6IC9ldGMva3ViZXJuZXRlcy9zc2wvYXBpc2VydmVyLWNydC5wZW0KICAgIGNsaWVudC1rZXk6IC9ldGMva3ViZXJuZXRlcy9zc2wvYXBpc2VydmVyLWtleS5wZW0KY2x1c3RlcnM6Ci0gbmFtZTogbG9jYWwKICBjbHVzdGVyOgogICAgY2VydGlmaWNhdGUtYXV0aG9yaXR5OiAvZXRjL2t1YmVybmV0ZXMvc3NsL2FwaXNlcnZlci1jYS5wZW0KICAgIHNlcnZlcjogaHR0cHM6Ly9BUElEb21haW4KY29udGV4dHM6Ci0gY29udGV4dDoKICAgIGNsdXN0ZXI6IGxvY2FsCiAgICB1c2VyOiBzY2hlZHVsZXIKICBuYW1lOiBzZXJ2aWNlLWFjY291bnQtY29udGV4dApjdXJyZW50LWNvbnRleHQ6IHNlcnZpY2UtYWNjb3VudC1jb250ZXh0Cg=="
 
-    - path: /etc/kubernetes/encryption/k8s-encryption-config.yaml
```

Comment:
This seems wrong - the test should be altered to still render the master. We can take a look on how to implement this. Just noting that this is now called master render but renders a worker :)